lib/src/widget/built_in/simple/simple.dart

```diff
@@ -77,32 +93,43 @@ class SimpleToastWidget extends StatelessWidget {
     required BorderSide borderSide,
     required BorderRadiusGeometry borderRadius,
     required bool applyBlurEffect,
+    required bool showCloseButton,
   }) {
-    Widget body = Container(
+    Widget body;
+
+    if (showCloseButton) {
+      body = Row(
+        children: [
+          Expanded(
+            child: widget.title ?? const SizedBox(),
+          ),
+          ToastCloseButton(
+            showCloseButton: showCloseButton,
+            onCloseTap: widget.onCloseTap,
+            icon: widget.defaultStyle.closeIcon(context),
+            iconColor: widget.foregroundColor?.withOpacity(.3) ??
+                widget.defaultStyle.closeIconColor(context),
+          ),
+        ],
+      );
+    } else {
```

Comment:
We don't need this else here becase `ToastCloseButton` will handle its own visibility